src/Symfony/Component/Debug/Exception/FlattenException.php

```diff
@@ -324,4 +327,33 @@ private function getClassNameFromIncomplete(\__PHP_Incomplete_Class $value)
 
         return $array['__PHP_Incomplete_Class_Name'];
     }
+
+    public function getTraceAsString()
+    {
+        return $this->traceAsString;
+    }
+
+    public function __toString()
```

Comment:
I'm good keeping getTraceAsString, but __toString is too risky IMHO, better not add it.